### ISSUE TITLE
Add keyjazzing to the `Open Instrument` dialog

### ISSUE THREAD

**Issue by FreeFull** [kometbomb/klystrack#204]
The feature is useful for previewing instruments before loading them, and is present in other software, for example sunvox.

**Comment by darkhog** [comment]
Yeah, it's really useful, as a fellow Sunvox user I can attest for that.
